### PR TITLE
Dbz 2372 - Readme demo review edits

### PR DESCRIPTION
**Pull request by roldanbob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,78 @@
...
-### Data Replication
+### Data replication
 
-The second scenario covers how to avoid dual writes. You have a legacy PHP application that needs to update an Elasticsearch index with the new orders captured for future search. The CDC part is covered with Debezium connectors and includes the use of Single Message Transformations The second part consists of the deployment of the new Camel Kafka Connectors.
+The data replication scenario demonstrates how to use Debezium to avoid dual writes. In the scenario, you have a legacy enterprise application built on PHP, that sends orders and records into a database. To enable faster querying of your orders, you want to  send the records for each new order event to an Elasticsearch index. 
 
-This is a basic demo where everything is already set up for you to go over the solution pattern from the solution explorer web application
+The scenario makes use of Debezium connectors and Single Message Transformations (SMTs). The second part of the demo introduces the new Camel Kafka Connectors.
+
+The components used in this scenario are deployed in the Solution Explorer, and are ready for you to use.
```
Comment (hguerrero) [inline README.md]: The components are deployed into the debezum-complete-demo namespace and the script to follow is in Solution Explorer.

File: README.md [hunk 2]
```diff
@@ -1,60 +1,78 @@
...
 
-### Products and Projects
+### Products and projects
 
 - Red Hat OpenShift
 - Red Hat Integration
   - Debezium
   - AMQ Streams
-- Camel Kafka Connectors
+- Apache Camel Kafka Connectors
```
Comment (hguerrero) [inline README.md]: Apache Camel Kafka Connect connectors

**Review by hguerrero — CHANGES_REQUESTED** [review]

**Comment by roldanbob** [comment]
Commit 081e087 addresses the requested changes and incorporates a few further wording changes aimed at clarifying content.

**Comment by hguerrero** [comment]
Thanks @roldanbob

**Pull request merged**